README.md

```diff
@@ -32,5 +32,40 @@ apps = client.list_apps()
 app = client.create_app(app_id="demo_app")
 dataset = app.create_dataset(dataset_id="demo_dataset")
 # execute data upload to Clarifai app dataset
-dataset.upload_dataset(task='visual_segmentation', split="train", dataset_loader='coco_segmentation')
+dataset.upload_dataset(task="visual_segmentation", split="train", dataset_loader="coco_segmentation")
+```
+
+## Interacting with Models
+
+### Model Predict
+```python
+from clarifai.client.model import Model
+
+# Model Predict
+model = Model(user_id="user_id", app_id="app_id", model_id="model_id")
+model_prediction = model.predict_by_url(url="url", input_type="image") # Supports image, text, audio, video
```

Comment:
How would this work for a mutli-modal model?

Reply:
We planned to add Input base class and users can directly use `predict` function for multi model inputs.